From f80d8336881844230143f21bbfd07dba6a309d25 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <quic_jouni@quicinc.com>
Date: Tue, 18 Jul 2023 16:08:44 +0300
Subject: [PATCH] KW: hostapd: ACS: Remove invalid debug print

ideal_chan is NULL here, so it is not really valid to try to debug print
something from it due to the implied NULL pointer dereferencing.

Fixes: af0f60e7dd00 ("EHT: Calculate puncturing bitmap for ACS")
Signed-off-by: Jouni Malinen <quic_jouni@quicinc.com>
---
 src/ap/acs.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/ap/acs.c b/src/ap/acs.c
index 877d7d40a..8b5760918 100644
--- a/src/ap/acs.c
+++ b/src/ap/acs.c
@@ -1076,12 +1076,6 @@ bw_selected:
 		return ideal_chan;
 	}
 
-#ifdef CONFIG_IEEE80211BE
-	if (iface->conf->punct_acs_threshold)
-		wpa_printf(MSG_DEBUG, "ACS: RU puncturing bitmap 0x%x",
-			   ideal_chan->punct_bitmap);
-#endif /* CONFIG_IEEE80211BE */
-
 	return rand_chan;
 }
 
-- 
2.34.1

